blog/content/post/2023-04-27-3mdeb-open-source-contribution.md

```diff
@@ -0,0 +1,225 @@
+the [meta-sunxi](https://github.com/linux-sunxi/meta-sunxi/pull/256) layer.
+
+By partnering with our team, we can help you leverage the power of Yocto and
+build a custom Linux distribution that fully aligns with your vision. From
+feature-rich IoT devices to mission-critical servers, we've got you covered.
+Let us create a personalized solution that meets your exact specifications
+and takes your device's capabilities to the next level.
+
+![Trenchboot logo](/img/trenchboot_logo.svg)
+
+[TrenchBoot](https://blog.3mdeb.com/tags/trenchboot/) is a framework that
+allows individuals and projects to build security engines to perform launch
+integrity actions for their systems. The framework builds upon Boot Integrity
+Technologies (BITs) that establish one or more Roots of Trust (RoT) from which
+a degree of confidence that integrity actions were not subverted is derived.
+
+The biggest changes took place in the landing-zone component. Worth mentioning
+is the addition of [support for the Xen hypervisor](https://github.com/TrenchBoot/landing-zone/pull/60)
+and [multiboot2 support for the GRUB2 bootloader](https://github.com/TrenchBoot/landing-zone/pull/64).
+The author of these changes is Krystian Hebel.
+
+If you're looking for methods of enhancing boot security for your devices,
+Trenchboot is the solution you've been searching for. Our team is equipped
+to enable Trenchboot for you so that you can enjoy enhanced protection against
+attacks and device security breaches. Let us help you safeguard your devices
+with Trenchboot.
+
+
+## Upcoming events
+
+Don't forget to mark your calendars for
+[Dasharo User Group #2](https://vpub.dasharo.com/e/7/dasharo-user-group-2),
+which will take place on July 6th, 2023. This is a great opportunity to learn
+more about open-source projects and connect with other members of the community.
+
+The Dasharo User Group (DUG) is an important forum for users of Dasharo to
+come together, share their knowledge, and stay informed about the latest
+developments in the Dasharo ecosystem. The DUG is a platform for users to
+connect and learn about new features and updates that are coming to Dasharo.
+The first DUG event will take place in early March and will include a variety
+of discussions on different topics related to Dasharo. The agenda for the event
+will be shared in the next month. The event will be a great opportunity for
+Dasharo users to meet other users, learn new things, and share their
+knowledge and experience with others.
+
+Dasharo vPub 0x7 is a follow-up event to DUG#2 and will provide a space
+to engage in more informal conversations and discussions that may not have
+been covered during DUG#2. The vPub is designed to be a less structured, more
+relaxed environment where the community can discuss topics that are of interest
+to them. This can include off-topic discussions, technical challenges they are
+facing, and ideas for new features or improvements.
```

Comment:
Why not link information about previous events, videos from `DUG#1`, and agenda from vPub 0x6?

Reply:
Links added. https://github.com/3mdeb/news-and-ideas/pull/519/commits/68a72cdebcee326365faf61a3b7861f718765122#diff-5dff579917e1df023c3d68f093f637f7fb497ee2f22650cc786ac30a195a7eceR358